react-sample/src/Counter.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react"
+import PropTypes from "prop-types"
+
+function Counter({ initialCount  } : { initialCount: number}) {
+  const [count, setCount] = useState(initialCount)
+
+  function incrementCount() {
+    setCount(count + 1)
+    console.log(count)
```

Comment:
**GPT COMMENT**: 
 - Avoid using console.log in your functions. It's generally used for debugging and not meant to be in production code.